src/cmd/cluster_intel_collector/builder.go

```diff
@@ -0,0 +1,126 @@
+package cluster_intel_collector
```

Comment:
Why avoid speaking package names just because they are a bit longer? Although I'm not completely convinced by `cluster_intel_collector` myself. just `support` or `archive` or `collector` are too generic for my taste. I'd vote for `support_archive`, also because it is (as mentioned in a comment below) an well established concept.

Reply:
so the problem is that the whole structure is bit off.

having a separate command for the current `troubleshoot` and the `collection` is what we want, they should probably be sub commands of a `support` or `troubleshoot` command

so how it would work in my head is that we have a `support` command (which would also the name of the package)
then there would be a sub package called `troubleshoot` where the current `troubleshoot` package should be moved
and the contents of this PR would be moved into an `archive` package
so the end result would look like:
```
/src/cmd/support
                    |
                    |- troubleshoot/
                    |                 |
                    |           troubleshoot.go
                    |            ....
                    |
                    |- archive/
                    |         |
                    |        archive.go
                    |         ....
```
and from the user’s perspective there would be a `support` command with sub commands like `archive` and `troubleshoot`
examples: `support archive` or `support troubleshoot`

this would allow us to combine/chain the commands, so the user/support person has to run a single command that under the hood uses multiple sub commands.
example: `support` command without subcommands would
- Run the `troubleshoot` subcommand, and store the result somewhere
- Run the log `archive` subcommand, which would also put in the archive the result of the  `troubleshoot` subcommand


This how I think of it at least. This should be discussed with the team, and make sure that we agree on a design that works for all

TLDR: The current solution feels very rushed, and I think we need to go back to the drawing board, if we want  to introduce a feature that we wish to support and improve on in the future.

Reply:
Ok, we can discuss this, there are some pitfalls if we want to combine those commands into one (which it essentially is, if we put it under one common subcommand). But that is definitely out of scope for this PR.